src/main/java/com/atlassian/db/replica/spi/jdbc/JdbcProtocol.java

```diff
@@ -0,0 +1,17 @@
+package com.atlassian.db.replica.spi.jdbc;
+
+public enum JdbcProtocol {
```

Comment:
Enums are hard to evolve (https://github.com/wyrzyk/java-api-design-checklist/blob/master/CHECKLIST.md#255-do-not-add-new-values-to-a-released-enumeration). 

It should not be an issue, as enums should not evolve (https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html).

> An enum type is a special data type that enables for a variable to be a set of predefined constants. 

But it seems we don't have a full list of current and the future JdbcProtocols here 😉.

Reply:
replaced enum with final class with a private constructor and static instances